src/main/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineInputStep.java

```diff
@@ -0,0 +1,3 @@
+package io.jenkins.plugins.pipelinegraphview.utils;
+
+public record PipelineInputStep(String message, String cancel, String id, String ok, boolean parameters) {}
```

Comment:
jackson with our current config can't serialise the parameters object (if not more) so just map what we need